.github/workflows/image-push-release.yml

```diff
@@ -0,0 +1,39 @@
+name: "Push images on release"
+on:
+  push:
+    tags:
+      - v*
+jobs:
+  build-and-push-image-sriov-cni:
+    runs-on: ubuntu-20.04
+    env:
+      IMAGE_NAME: ghcr.io/${{ github.repository }}
+    steps:
+      - name: Check out the repo
+        uses: actions/checkout@v2
+
+      - name: Set up Docker Buildx
```

Comment:
are we using buildx to support multi arch?  if yes were that arch defined?

Reply:
We don't use it right now. It's done as same as for sriov-cni and other k8snetworkplumbingwg's projects to have the same approach over the whole organization